src/Microsoft.DotNet.Interactive.Formatting.Tests/Microsoft.DotNet.Interactive.Formatting.Tests.csproj

```diff
@@ -14,6 +14,7 @@
 
   <ItemGroup>
     <PackageReference Include="FluentAssertions" Version="5.10.0" />
+    <PackageReference Include="Assent" Version="1.3.1" />
```

Comment:
add
 <NoWarn>$(NoWarn);8002</NoWarn><!-- Assent isn't strongly signed --> to fix the build